src/site/content/en/blog/media-session/index.md

```diff
@@ -390,6 +400,79 @@ Resetting the position state is as easy as setting it to `null`.
 navigator.mediaSession.setPositionState(null);
 ```
 
+## Video conferencing actions {: #video-conferencing-actions }
+
+When the user puts their video call into a Picture-in-Picture window, the
+browser may display controls for the microphone, camera, and for hanging up.
+When the user clicks those, the website handles them through the video
+conferencing actions below.
+
+<figure class="w-figure">
+  {% Img
+    src="image/vvhSqZboQoZZN9wBvoXq72wzGAf1/fXc7jqc95Oa6sKce7kpZ.jpg",
+    alt="Screenshot of video conferencing controls in a Picture-in-Picture window",
+    width="748",
+    height="464"
+  %}
+  <figcaption class="w-figcaption">Video conferencing controls in a Picture-in-Picture window</figcaption>
```

Comment:
```suggestion
  <figcaption class="w-figcaption">Video conferencing controls in a Picture-in-Picture window.</figcaption>
```

Reply:
I always forget to look for the periods at the ends of figcaption and alt. Please double-check your whole article.

Reply:
Done for the all article